src/components/SnoozeItem.js

```diff
@@ -91,7 +94,7 @@ function SnoozeItem({ user, snooze, onDelete }) {
           color={theme.palette.secondaryLight.main}
           variant="subtitle1"
         >
-          Scheduled for {getFormattedDate(notifyAt)}
+          {notifyLabel} {getFormattedDate(notifyAt)}
```

Comment:
As both `notifyLabel` and `formattedDate` depends on `notifyAt`, what if you extract a dedicated component to do all the related operations?

Reply:
Yepp, I will look into this. I was also thinking to use a default value for the dateHasPassed but I went against it because the purpose of that function is to compare 2 dates. I think it's better to have the 2 param option to let the user know that a comparison between 2 dates is happening.

Reply:
Done